src/components/unrecoverable-error.njk

```diff
@@ -22,14 +22,21 @@
   <div>
       <p><a href="https://signin.account.gov.uk/contact-us" class="govuk-link" rel="noreferrer noopener" target="_blank">{{ translate("error.unrecoverable.contactMeLink") }}</a></p>
   </div>
-
+  <script {% if cspNonce %} nonce="{{ cspNonce }}"{%  endif %}>
+    window.addEventListener('load', function () {
+      window
+      .DI
+      .analyticsGa4
+      .pageViewTracker
+      .trackOnPageLoad({
+        statusCode: '500', // Access status code
+        englishPageTitle: '{{ translate("error.unrecoverable.title") }}',
```

Comment:
This won't translate to English if the language is set to Welsh, as the `translate` function is bound to the locale.

Reply:
Yes, we know that. this parameter is called "englishPageTitle" because after we push the analytics package in production (end of March?), data and analytics team will ask us to find a way to always send/use the english version of any text (e.g: page title, input label, error message).

Reply:
Sorry, what I meant was, I think this will translate in the chosen locale (e.g. Welsh), not in the default of English.

Reply:
We're expecting it to that currently as we don't have a way to send through english analytics info for welsh pages yet. It's an improvement that will be requested from the analytics team in the future but the expected behaviour now is that we send the page title through as `englishPageTitle` regardless of the translation as it's the structure requested by the analytics team

Reply:
Sounds like you're way ahead of this than I am :)